tests/test_utils.py

```diff
@@ -610,25 +610,25 @@ def tearDown(self):
         """Delete workspace."""
         shutil.rmtree(self.workspace_dir)
 
-    def test_build_lookup_from_csv(self):
-        """utils: test build_lookup_from_csv."""
+    def test_read_csv_to_dataframe(self):
+        """utils: test the default behavior"""
         from natcap.invest import utils
-        table_str = 'a,b,foo,bar,_\n0.0,x,-1,bar,apple\n'
-        table_path = os.path.join(self.workspace_dir, 'table.csv')
-        with open(table_path, 'w') as table_file:
-            table_file.write(table_str)
-        result = utils.build_lookup_from_csv(
-            table_path, 'a', to_lower=True)
-        expected_dict = {
-            0.0: {
-                'a': 0.0,
-                'b': 'x',
-                'foo': -1.0,
-                'bar': 'bar',
-                '_': 'apple'
-            },
-        }
-        self.assertDictEqual(result, expected_dict)
+
+        csv_file = os.path.join(self.workspace_dir, 'csv.csv')
+
+        with open(csv_file, 'w') as file_obj:
+            file_obj.write(textwrap.dedent(
+                """
```

Comment:
I see you have a `.strip()` there at the end (which is nice!) so this isn't a request to change anything, just an FYI!  I recently learned that you can also do this to avoid the leading newline:

```
"""\
HEADER,
A,
b
"""

Reply:
That is much nicer, I made this change everywhere in `test_utils.py`